front/lib/api/assistant/actions/utils.ts

```diff
@@ -64,4 +65,11 @@ export const ACTION_SPECIFICATIONS: Record<
     dropDownIcon: PlanetIcon,
     flag: null,
   },
+  VISUALIZATION: {
+    label: "Visualization",
+    description: "Generate graphs to visually represent the data",
```

Comment:
Generate graphs to visualize your data.
```suggestion
    description: "Generate graphs to visually represent the data",
```

Reply:
Applied!